app/Console/Kernel.php

```diff
@@ -2,11 +2,15 @@
 
 namespace App\Console;
 
+use App\Libraries\Slack;
+use Closure;
 use Illuminate\Console\Scheduling\Schedule;
 use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
 
 class Kernel extends ConsoleKernel
 {
+    private $failureEmail = 'privileged-access@vatsim-uk.co.uk';
```

Comment:
@AxonC Just realised this - could you update the email to be .uk rather than -uk.co.uk?